org.adishatz.Screenshot.json

```diff
@@ -0,0 +1,56 @@
+{
+    "app-id" : "org.adishatz.Screenshot",
```

Comment:
How do I know you own adishatz.org ? The webpage there is 403 https://adishatz.org/

Please put something up there like a link to https://gitlab.gnome.org/gnumdk/screenshot.git

Reply:
I already have an application with my domain at https://flathub.org/apps/org.adishatz.syncpasswd

Reply:
That's not verified so it doesn't  prove ownership

Reply:
Thanks!